cmake/GenerateBuiltins.cmake

```diff
@@ -503,3 +503,125 @@ function (generate_builtins)
         ${DISPATCH_BUILTIN_CPP_FILES} ${COMMON_BUILTIN_CPP_FILES} ${TARGET_BUILTIN_CPP_FILES})
     add_dependencies(builtin builtins-cpp)
 endfunction()
+
+function (generate_generic_target_builtin_1 ispc_name target arch bit os)
```

Comment:
Why the name has suffix `_1`?

Also it wouldn't hurt to document the function. I know, other functions are not documented, but I always struggled with understanding what they do without comments.

Reply:
I think I had initially two nested ones. Anyway, I have removed the suffix.